src/main/java/com/paneedah/mwc/bases/ItemBase.java

```diff
@@ -0,0 +1,9 @@
+package com.paneedah.mwc.bases;
+
+import net.minecraft.item.Item;
+
+public class ItemBase extends Item {
+    public ItemBase(){
```

Comment:
Missing space between class init and actuall stuff

Reply:
✅